partner_multi_relation/views/res_partner.xml

```diff
@@ -21,25 +21,13 @@
             <field name="model">res.partner</field>
             <field type="xml" name="arch">
                 <xpath expr="//div[@name='button_box']" position="inside">
-                    <button
-                        class="oe_inline oe_stat_button"
-                        type="action"
-                        context="{
-                            'search_default_this_partner_id': active_id,
-                            'default_this_partner_id': active_id,
-                            'active_model': 'res.partner',
-                            'active_id': id,
-                            'active_ids': [id],
-                            'active_test': False,
-                        }"
-                        name="%(action_res_partner_relation_all)d"
-                        icon="fa-users"
-                        >
-                        <field
```

Comment:
Leave the field element lines as is, there is nothing wrong with the existing formatting and indentation.